recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ build:
 
 requirements:
   build:
-    - {{ compiler('cgo') }}
+    - {{ compiler('cgo') }} =1.15.*
```

Comment:
This isn't the correct way to pin this. Adding it to `recipe/conda_build_config.yaml` with `cgo_compiler_version: 1.15` is the correct way. Otherwise if we introduce a global pinning for cgo, this would stop working.